envconfig/config.go

```diff
@@ -183,6 +183,8 @@ var (
 	NewEngine = Bool("OLLAMA_NEW_ENGINE")
 	// ContextLength sets the default context length
 	ContextLength = Uint("OLLAMA_CONTEXT_LENGTH", 4096)
+	// UseAuth enables authentication between the Ollama client and server
+	UseAuth = Bool("OLLAMA_USE_AUTH")
```

Comment:
```suggestion
	UseAuth = Bool("OLLAMA_AUTH")
```

nit on potentially a more consistent name since we don't have `_USE_` anywhere else